src/elements/management/plan.md

```diff
@@ -0,0 +1,15 @@
+# Management Plan
```

Comment:
This is a lot of pages. I'm wondering if some of them could be combined? I'm noticing some redundancy as carryover from how the previous site came together. Maybe Team + Governance and then Selection + Outreach + Adopt?

Reply:
Agreed that this needs consolidation for clarity.  Let's update in a future pull request.